src/app/upgrade.h

```diff
@@ -106,26 +118,62 @@ bool upgradeResumeFile(const QString &filepath)
 
 bool upgrade(bool ask = true)
 {
+    QIniSettings *oldResumeSettings = new QIniSettings("qBittorrent", "qBittorrent-resume");
+    QString oldResumeFilename = oldResumeSettings->fileName();
+    QVariantHash oldResumeData = oldResumeSettings->value("torrents").toHash();
+    delete oldResumeSettings;
+    if (oldResumeData.isEmpty()) {
+        Utils::Fs::forceRemove(oldResumeFilename);
+        return true;
```

Comment:
Don't return here. The .ini file might be missing or corrupt or whatever. If there are *.fastresume files they should be converted regardless.